app/plot/[repo]/[collection]/[plotName]/compare/[repo2]/[collection2]/page.js

```diff
@@ -0,0 +1,104 @@
+
+import React from 'react';
+import Link from 'next/link'
+
+import { GetSummary } from '@/lib/summaries'
+
+import DualPlotPager from '@/components/dualPlotPager'
+import PlotDisplay from '@/components/plotDisplay'
+
+export const revalidate = 180
+
+export default async function Collection({params, searchParams}) {
+
+
+    const findPlotEntries = (collectionData, plotName)  => {
+
+        const tractEntries = collectionData['tracts']?.[plotName] ?? []
+        const visitEntries = collectionData['visits']?.[plotName] ?? []
+        const globalEntries = collectionData['global']?.[plotName] ?? []
+
+        return [tractEntries, visitEntries, globalEntries].flat()
+    }
+
+    const repo = decodeURIComponent(params['repo'])
+    const repo2 = decodeURIComponent(params['repo2'])
+    const collection = decodeURIComponent(params['collection'])
+    const collection2 = decodeURIComponent(params['collection2'])
+    const plotName = decodeURIComponent(params['plotName'])
+
+    const currentPage = parseInt(searchParams?.page) ? parseInt(searchParams?.page) : 1
+
+    const collectionData = await GetSummary(repo, collection)
+    const collectionData2 = await GetSummary(repo2, collection2)
+
+    const dataIdSortFunc = (a, b) => {
+        const dataIdA = JSON.parse(a.dataId)
+        const dataIdB = JSON.parse(b.dataId)
+        const bandsOrder = ['u', 'g', 'r', 'i', 'z', 'y']
+
+        if('tract' in dataIdA && 'tract' in dataIdB) {
+            const tract_comparision = dataIdA.tract - dataIdB.tract
```

Comment:
Is `tract_comparision` a typo for `tract_comparison`? i.e., without the second "i".